common/views/components/PageHeader/PageHeader.tsx

```diff
@@ -44,7 +63,9 @@ const HeroPictureBackground = styled.div<{ bgColor: PaletteColor }>`
     )}
 `;
 
-const HeroPictureContainer = styled.div`
+const HeroPictureContainer = styled.div.attrs({
```

Comment:
would this hide the hero image in print (and is there not a good chance that we'd want that to be part of e.g. a visual story)?